misc/10-aws.conflist

```diff
@@ -15,7 +15,7 @@
     {
       "name": "egress-cni",
       "type": "egress-cni",
-      "mtu": "9001",
```

Comment:
Thanks for the contribution, @benben .

This change can covered by a test case addition to 
https://github.com/aws/amazon-vpc-cni-k8s/blob/master/cmd/aws-vpc-cni/main_test.go#L51

looks like we didn't cover this initially when we had the opportunity to do so. Please see if you can add a test case for this in the above linked file.

Reply:
We will bring this test separately.